### PR TITLE
Updated Auto Kick/Ban Logging

### PR DESCRIPTION
**Pull request by shashankpandey04**

### CODE REVIEW

**Review by NoahCxrest — APPROVED** [review]
Approved

File: cogs/ActivityNotices.py
Comment (MikeyUsersREC) [inline cogs/ActivityNotices.py:1]: **[ DO NOT PUSH ]**
RA Requests will not work with this change.

Comment (MikeyUsersREC) [inline cogs/ActivityNotices.py:1]: **[ DO NOT PUSH ]**
RA Requests will not work with this change.

File: cogs/CustomCommands.py
Comment (MikeyUsersREC) [inline cogs/CustomCommands.py:1]: Line 325-330 doesn't fix the underlying issues if ['message'] is none.

File: cogs/Punishments.py
Comment (MikeyUsersREC) [inline cogs/Punishments.py:1]: "Management" here refers to a shorthand of "Punishment Management", not "Management" as a permission level.

File: erm.py
Comment (MikeyUsersREC) [inline erm.py:1]: I have no problem with line 154.

713-740 **should not** be in the main erm file. This is meant to be a file to be loaded before everything else. Specific utilities should be modularised in the utils folder. `utils/utils.py` would be a better destination. Also change the name of the `run_command` function here to something more related to its scope - as it'd be confused with `PRCApiClient.run_command`

File: menus.py
Comment (MikeyUsersREC) [inline menus.py:1]: I have some problems with Line 6905, and similar.

Data schemes should always be present in the dataset. We shouldn't have values that don't exist, as that would make our data schema less clean. All values should be present, as with most of the newer modules.

**Review by MikeyUsersREC — CHANGES_REQUESTED** [review]
See comments for details.

**Pull request closed without merging**